stubs/Symfony/Component/Form/AbstractType.stub

```diff
@@ -4,27 +4,28 @@ namespace Symfony\Component\Form;
 
 /**
  * @template TData
+ * @template TOptions of array<string, mixed>
```

Comment:
This has to be optional, otherwise it's a BC break. See https://github.com/phpstan/phpstan-src/pull/3457